app/views/katello/api/v2/subscriptions/base.json.rabl

```diff
@@ -16,3 +16,10 @@ attributes :name => :product_name
 attributes :unmapped_guest
 attributes :virt_only
 attributes :virt_who
+
```

Comment:
It looks like this may cause troubles for activation keys.  In setting up a test of the updates, attempting to add a subscription to a key in the UI generates the following error:
```
2017-11-17T14:17:27 f8cfb8c9 [app] [E] NoMethodError: undefined method `hypervisor' for #<Katello::Subscription:0x007fe42e4e7770>
```
Stack: https://paste.fedoraproject.org/paste/srvlp53-t6f2z-nxfBlTnQ